ibis/backends/bigquery/tests/unit/udf/test_usage.py

```diff
@@ -49,6 +49,23 @@ def my_len(s):
     snapshot.assert_match(sql, "out.sql")
 
 
+def test_udf_sql(snapshot):
+    _udf_name_cache.clear()
+
+    format_t = udf.sql(
+        "format_t",
+        params={'input': dt.string},
+        output_type=dt.double,
+        sql_expression="[FORMAT('%T', input)",
```

Comment:
Is this meant to test an error? It looks like a stray `[` may be in there!

Reply:
Fixed.

Reply:
Thanks!